pkg/tsdb/elasticsearch/healthcheck.go

```diff
@@ -22,59 +23,56 @@ func (s *Service) CheckHealth(ctx context.Context, req *backend.CheckHealthReque
 		logger.Error("Failed to get data source info", "error", err)
 		return &backend.CheckHealthResult{
 			Status:  backend.HealthStatusUnknown,
-			Message: "Failed to get data source info",
-		}, err
```

Comment:
Had to stop passing the err here and instead set it to null becuase otherwise the error message wasn't being surfaced to the user.